### ISSUE TITLE
Ruby config.yml Instructions

### ISSUE THREAD

**Issue by RamblingCookieMonster** [juthilo/run-jekyll-on-windows#39]
In case it helps - after the ```ruby dk.rb init``` step, it might be worth noting that the user should add the path to the Ruby installation to their config.yml (it is not performed automatically, at least on the systems I have tested this on).

Thanks for the handy instructions!

**Comment by juthilo** [comment]
To clarify, you checked the "Add Ruby executables to your PATH" box during setup, right? (Or kept it checked.)

Could this be affected by whether or not the installation is run with admin permissions?

**Comment by RamblingCookieMonster** [comment]
Yes, path includes the Ruby dir - the config.yml generated by ruby dk.rb init is just empty.

Should have searched before - It looks like it might be a [known issue](http://stackoverflow.com/questions/16523607/cant-get-ruby-devkit-configuration-file-autogenerated-properly)?

**Comment by juthilo** [comment]
Oops, I totally misread the issue as being about PATH missing Ruby after installation. Sorry!

I'm gonna check out your link and probably add a note about this to the guide soon. Seems like 64bit Windows can be a real deal breaker for Ruby these days in general...

**Comment by RamblingCookieMonster** [comment]
No worries!

Apart from that one bit, everything worked flawlessly on both Windows 10 and Windows 8.1.

**Comment by dlongest** [comment]
I followed the guide today on Windows 7 64-bit, had the same issue with having to manually set the dev kit path in config.yml after running dk.rb init. I was running the installer as an administrator at the time.  A small note in the guide would be sufficient IMO,  the error message from running dk.rb install is pretty clear but would be nice to point it out since it seems pretty well established. Great work on the guide, much easier then the last time I fought through it on my own.

**Comment by clintstotesbery** [comment]
This is a 64-bit specific issue with the DevKit. #53 fixes this in bcf23f3 by documenting the workarounds.
